### ISSUE TITLE
Figure out how we want to add a link to the audio described version of videos

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1811]
Everything highlighted in yellow is a video in the Production troubleshooting guide.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/0a8bc6b7-fe4c-4a46-8e1c-60a757b1cc22)

**Comment by mcharg** [comment]
Accordion to digital.gov, as long as we provide a link to an audio described version in the video description, we should be good. @Maroyafaied Are you good with using this strategy?

![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/c3bc7a4a-2d3b-41f5-8a6d-d4b9c43ec467)
https://digital.gov/2014/06/30/508-accessible-videos-why-and-how-to-make-them/

**Comment by mcharg** [comment]
Or we could do something like this. ![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/7f363302-a851-41cc-b6cf-cd6d8257c32d)
